src/crimson/osd/pg.cc

```diff
@@ -314,15 +314,15 @@ void PG::do_delete_work(ceph::os::Transaction &t)
   shard_services.dec_pg_num();
 }
 
-void PG::scrub_requested(bool deep, bool repair, bool need_auto)
+void PG::scrub_requested(scrub_level_t scrub_level, scrub_type_t scrub_type)
```

Comment:
the title of the commit message reads

> osd: extracting scrub-related code from the PG (8 of n)

but it fails to summarize the change made in this commit. i'd suggest change it to something like

> crimson/osd: adapt to the scrub refactory in classic osd